localflavor/kw/kw_governorates.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.utils.translation import ugettext_lazy as _
+
+# List of governorates as defined by ISO_3166-2
+# For reference; see https://en.wikipedia.org/wiki/ISO_3166-2:KW
+# Note that these are governorates and not subdivisons
```

Comment:
Governorates are different than subdivisions, in that subdivisions don't have governors, and governorates do.

So, in Kuwait there is a Governor of Ahmadi

The spelling mistake, I'll fix that ASAP.